src/precompute_ecmult.c

```diff
@@ -77,15 +79,15 @@ int main(void) {
     fprintf(fp, "#include \"ecmult.h\"\n");
```

Comment:
`ecmult.h` is no longer needed if you remove `ECMULT_TABLE_SIZE`.

Reply:
I think it's correct to use `ECMULT_TABLE_SIZE`.

Reply:
Right sorry.  For some reason I thought all the users of `ECMULT_TABLE_SIZE` were eliminated, but that is not true.